database/postgres/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+VERSION="1.24.0"
+NAMESPACE="postgres"
+
+# Install cloudnative-pg operator.
+kubectl apply --server-side -f \
+  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/main/releases/cnpg-${VERSION}.yaml
+kubectl rollout status deployment --namespace=cnpg-system --timeout=90s
+
+# Create the postgres database server.
+kubectl create ns ${NAMESPACE}
+kubectl -n ${NAMESPACE} apply -f .
+kubectl -n ${NAMESPACE} wait pod/kubearchive-1 --for=create --timeout=60s
+kubectl -n ${NAMESPACE} wait pod --all --for=condition=ready --timeout=30s
+
+# Create the kubearchive database
+LOCAL_PORT=5433
+echo Forwarding port ${LOCAL_PORT} to service/kubearchive-wr:5432.
+export PGPASSWORD=$(kubectl -n ${NAMESPACE} get secret kubearchive-superuser -o jsonpath='{.data.password}' | base64 --decode)
+kubectl -n ${NAMESPACE} port-forward service/kubearchive-rw ${LOCAL_PORT}:5432 >& /dev/null &
+pid=$!
```

Comment:
You don't need to register the `pid` if you then kill all background jobs.

Reply:
Fixed.